src/Symfony/Bundle/SecurityBundle/Templating/Helper/LogoutUrlHelper.php

```diff
@@ -68,7 +72,7 @@ public function registerListener($key, $logoutPath, $csrfTokenId, $csrfParameter
      *
      * @return string The logout path
      */
-    public function getLogoutPath($key)
+    public function getLogoutPath($key = null)
```

Comment:
@xabbuh The `getLogoutPath()` and `getLogoutUrl()` from the helper can indeed used with a default argument and do not have to change. 

Only the `getLogoutPath()` and `getLogoutUrl()` from the `LogoutUrlExtension` must have a default value, which according to the BC promise is allowed. The only question I have, in which UPGRADE file must I add this information?